jenkins/build-pod.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+  - name: py36
+    image: circleci/python:3.6
+    resources:
+      requests:
+        cpu: 1
+    command:
+    - cat
+    tty: true
+    securityContext:
+      runAsUser: 0
+      privileged: true
+  - name: rpmbuild
+    image: rpmbuild/centos7
+    resources:
+      requests:
+        cpu: 1
+    command:
+    - cat
+    tty: true
+    securityContext:
+      runAsUser: 0
+      privileged: true
+  - name: py27
+    image: circleci/python:2.7
+    resources:
+      requests:
+        cpu: 1
+    command:
+    - cat
+    tty: true
+    securityContext:
+      runAsUser: 0
+      privileged: true
+  - name: debbuild
+    image: debian:9
+    command:
+    - cat
+    tty: true
+    securityContext:
+      runAsUser: 0
+      privileged: true
+  nodeSelector:
+  instance-type: spot-xlarge
```

Comment:
how much time this flow runs on xlarge? and on large instance?

Reply:
For the flow it takes about the same time, maybe a few seconds difference, the big difference is in creating the pod, with large instance it can take a while for it to load the pod, while with xlarge it's definitely quicker.
I changed it to xlarge mainly to have faster builds to test my changes faster.
I'll change and keep it on large, so it won't consume unneeded resources.

Reply:
Sounds good